docs/UserGuide.md

```diff
@@ -358,8 +358,11 @@ Location: Hougang
 ### Editing an appointment: `edit_appointment`
 
 Edits an appointment with a specific index. Only the attributes present are changed in the appointment.
+If any one of the following three fields (DATE, TIME_FROM, TIME_TO), then all three 
+must also be present.
 
-Format: `edit_appointment INDEX [n/NAME] [s/SUBJECT_NAME] [d/DATE] [fr/TIME_FROM] [to/TIME_TO] [l/LOCATION]`
+Format: `edit_appointment [index] [n/name] [s/SUBJECT_NAME] [d/DATE] [fr/TIME_FROM] 
```

Comment:
I think that [index] is incorrect. The square brackets indicate that an argument is optional, which it is not here. Furthermore, index and name should be uppercase

Reply:
It should be `INDEX [n/NAME]`